android/app/src/main/java/app/organicmaps/editor/EditorFragment.java

```diff
@@ -469,6 +469,16 @@ private void initViews(View view)
     mDetailsBlocks.put(Metadata.MetadataType.FMD_PHONE_NUMBER, blockPhone);
     mDetailsBlocks.put(Metadata.MetadataType.FMD_CUISINE, blockCuisine);
     mDetailsBlocks.put(Metadata.MetadataType.FMD_INTERNET, blockWifi);
+    mDetailsBlocks.put(Metadata.MetadataType.FMD_WEBSITE, websiteBlock);
+    mDetailsBlocks.put(Metadata.MetadataType.FMD_WEBSITE_MENU, websiteMenuBlock);
+    mDetailsBlocks.put(Metadata.MetadataType.FMD_WEBSITE_MENU, websiteMenuBlock);
+    mDetailsBlocks.put(Metadata.MetadataType.FMD_EMAIL, emailBlock);
+    mDetailsBlocks.put(Metadata.MetadataType.FMD_CONTACT_FACEBOOK, facebookContactBlock);
+    mDetailsBlocks.put(Metadata.MetadataType.FMD_CONTACT_INSTAGRAM, instagramContactBlock);
```

Comment:
How did it work before?

Reply:
Previously, it ignored the configuration described in editor.config. You need to add the field as shown here to make it take this configuration into account.

![Screenshot_20240519-084611.png](https://github.com/organicmaps/organicmaps/assets/116185937/8e39b5e6-29da-4afb-9e4f-d41714980354)

Reply:
@biodranik Do I need to change anything?